diff --git a/Users/tdurieux/git/dinghy-experiment/data/evaluation/dockerfiles/pyth-network/pyth-client/docker/fuzz/Dockerfile b/Users/tdurieux/git/dinghy-experiment/data/evaluation/dockerfile_repair_results/pyth-network/pyth-client/docker/fuzz/Dockerfile/repaired.Dockerfile
index 69b6b2d..59fc787 100644
--- a/Users/tdurieux/git/dinghy-experiment/data/evaluation/dockerfiles/pyth-network/pyth-client/docker/fuzz/Dockerfile
+++ b/Users/tdurieux/git/dinghy-experiment/data/evaluation/dockerfile_repair_results/pyth-network/pyth-client/docker/fuzz/Dockerfile/repaired.Dockerfile
@@ -1,8 +1,7 @@
 FROM ubuntu:20.04
 
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update
-RUN apt-get install -qq cmake curl git libzstd1 libzstd-dev python3-pytest sudo zlib1g zlib1g-dev libssl-dev clang llvm
+RUN apt-get update && apt-get install -y --no-install-recommends -qq cmake curl git libzstd1 libzstd-dev python3-pytest sudo zlib1g zlib1g-dev libssl-dev clang llvm && rm -rf /var/lib/apt/lists/*;
 
 # Grant sudo access to pyth user
 RUN echo "pyth ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers